packages/shared/src/graphql/sources.ts

```diff
@@ -82,6 +82,7 @@ export interface Source {
   headerImage?: string;
   color?: string;
   description?: string;
+  createdAt?: Date;
```

Comment:
Probably not exposed for this one right?
We can just query based on it and not return it.

Reply:
yeah true we won't need to expose it, just query by it would be enough